### PR TITLE
Added comment partial

### PR DESCRIPTION
**Pull request by Jonathan6**

### CODE REVIEW

File: views/partials/comment.handlebars [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
-  {{comment}}
-</div>
-<div class="mui--text-caption">
-  {{user_id}}
-  *
-  {{format_date updatedAt}}
-</div>
-<div class="mui-divider"></div>
\ No newline at end of file
+<div class="card bg-secondary mb-3">
```
Comment (ac524) [inline views/partials/comment.handlebars:1]: `bg-secondary` makes the text a bit too hard to read. I think `bg-light` would be more appropriate here.

**Review by ac524 — CHANGES_REQUESTED** [review]

**Comment by Jonathan6** [comment]
Accidentally added a js file I'm still working on. If there is a way to remove it from this pull request please let me know.

**Review by ac524 — APPROVED** [review]

**Comment by ac524** [comment]
I got you @Jonathan6! It takes a few steps, so bit hard for me to explain here. Would be a great question for office hours :) I reapplied the small change. You'll likely need to pull and do a small merge.

**Comment by ac524** [comment]
I'll let you complete the merge and delete branch step @Jonathan6 so you can make sure you get you pull request in first to grab my one new commit for your local history.

**Pull request merged**